CMakeLists.txt

```diff
@@ -27,9 +27,8 @@ file(GLOB_RECURSE SOURCES_LIB lib/*.cpp)
 add_subdirectory(vendor/uvw)
 
 add_executable(service ${SOURCES_SERVICE})
-find_library(filesystem_lib NAMES stdc++fs)
 find_library(sqlite3_lib NAMES sqlite3)
-target_link_libraries(service PUBLIC uv_a ${filesystem_lib} ${sqlite3_lib})
+target_link_libraries(service PUBLIC uv_a ${sqlite3_lib})
```

Comment:
I'm not sure, but I think you may still need some of these options. TBH I wouldn't mind manual checks for the compiler version as a temporary fix.

Reply:
Seem's fix since gcc 8.2 !